permissions_validators/src/public_blockchain/mod.rs

```diff
@@ -80,9 +80,12 @@ pub fn default_permissions() -> InstructionJudgeBoxed {
             .with_validator(key_value::GrantMyAssetDefinitionSet.into_validator())
             .with_validator(key_value::GrantMyAssetDefinitionRemove.into_validator())
             .no_denies()
-            .build();
+            .no_display_operation()
+            .build()
+            .into_validator()
+            .display_as("Grant validator".to_owned());
```

Comment:
I don't like implicit allocations, but in this place I can even do it to be `&'static str`, because this name is given at compile-time and won't be changed